pyproject.toml

```diff
@@ -67,7 +68,11 @@ select = [
   "T201",  # print
   "C4",  # flake8-comprehensions
   "RUF",  # Ruff-specific rules
-  "PL"  # pylint
+  "PL",  # pylint
+  "PD",  # pandas-vet
+  "UP",  # pyupgrade
+  "PERF",  # perflint
+  "FURB"  # refurb
```

Comment:
should we note what they do, so it's easier to see at a glance? I had to look them up one by one

Reply:
Good idea, done in [63d4a28](https://github.com/mckinsey/vizro/pull/813/commits/63d4a28b8455d14d2b96408db4149ebd34791cde).